packages/peregrine/lib/talons/Region/useRegion.js

```diff
@@ -48,16 +48,17 @@ export const useRegion = props => {
     }, [country, regionInputFieldApi, regionSelectFieldApi]);
 
     const { data, error, loading } = useQuery(getRegionsQuery, {
-        variables: { countryCode: country }
+        variables: { countryCode: country },
+        skip: !country
     });
 
     let formattedRegionsData = [{ label: 'Loading Regions...', value: '' }];
-    if (!loading && !error) {
+    if (data && !loading && !error) {
         const { country } = data;
         const { available_regions: availableRegions } = country;
         if (availableRegions) {
             formattedRegionsData = availableRegions.map(region => ({
-                key: region.id,
+                key: region.key,
```

Comment:
I don't believe `key` exists on this type, should probably change back.

```suggestion
                key: region.id,
```